packages/libs/client/src/signing/tests/signWithChainweaver.test.ts

```diff
@@ -6,20 +6,18 @@ jest.mock('cross-fetch', () => {
 });
 import { IPactCommand } from '../../interfaces/IPactCommand';
 import { ICommandBuilder, Pact } from '../../pact';
-import {
-  IChainweaverSignedCommand,
-  signWithChainweaver,
-} from '../signWithChainweaver';
+import { IQuicksignResponse } from '../../signing-api/v1/quicksign';
+import { signWithChainweaver } from '../signWithChainweaver';
 
 import fetch from 'cross-fetch';
 
 describe('signWithChainweaver', () => {
   jest.setTimeout(1000);
 
-  it('makes a call on 127.0.0.1:9467/v1/quickSign with transaction', async () => {
+  it('makes a call on 127.0.0.1:9467/v1/quicksign with transaction', async () => {
     (fetch as jest.Mock).mockResolvedValue({
       status: 200,
-      text: () => JSON.stringify({ results: [] }),
+      text: () => JSON.stringify({ responses: [] }),
```

Comment:
We can use the types from the generated openapi spec to define the response

Eg 

```suggestion
      text: () => JSON.stringify({ responses: [] } as IQuicksignResponse),
```

Reply:
I've added `IQuicksignResponseBody` interface